runner/server_workspace_blob.go

```diff
@@ -33,14 +41,48 @@ func (r *TerraformRunnerServer) CreateWorkspaceBlob(ctx context.Context, req *Cr
 		return nil, err
 	}
 
+	secretName := "tf-runner.cache-encryption"
```

Comment:
Shouldn't this match the secret name we're adding in the Helm templates of this PR i.e. `tf-runner-encryption`?

Reply:
fixed in the chart, as I prefer `tf-runner.cache-encryption` over `tf-runner-encryption`, that describes much better the purpose of that key.

Reply:
and follows the naming of the mtls secret:
```
terraform-runner.tls-1703067430   kubernetes.io/tls                     4      28h
```